index.js

```diff
@@ -398,6 +397,12 @@ instance.prototype.updateScenesAndSources = async function() {
 		self.scenes[scene.name] = scene;
 	}
 
+	if (self.states['studio_mode'] = true) {
```

Comment:
This would need to be an `==` or `===`

Reply:
Thank you for catching, fixed.

Reply:
please re-test

Reply:
I re-tested upon change, and it still works as expected